src/errors/runtime/message.js

```diff
@@ -13,9 +13,11 @@ export default {
     portIsNotFree:                      'The specified {portNum} port is already in use by another program.',
     invalidHostname:                    'The specified "{hostname}" hostname cannot be resolved to the current machine.',
     cantFindSpecifiedTestSource:        'Cannot find a test source file at "{path}".',
+    cannotParseRawFile:                 'Cannot parse a test source file due to an error.\n\n {errMessage}',
     cannotPrepareTestsDueToError:       'Cannot prepare tests due to an error.\n\n {errMessage}',
     fixtureNameIsNotAString:            'The fixture name is expected to be a string, but it was "{type}".',
     fixturePageIsNotAString:            'The page URL is expected to be a string, but it was "{type}".',
     testNameIsNotAString:               'The test name is expected to be a string, but it was "{type}".',
-    testBodyIsNotAFunction:             'The test body is expected to be a function, but it was "{type}".'
+    testBodyIsNotAFunction:             'The test body is expected to be a function, but it was "{type}".',
+    attemptToRunSeveralCommands:        'Attempt to execute a command when the previous one is still executing is detected'
```

Comment:
An attempt to execute a command when a previous command is still being executed was detected.